App.tsx

```diff
@@ -1,20 +1,114 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { StatusBar } from "expo-status-bar";
+import { useState } from "react";
+import {
+	StyleSheet,
+	View,
+	FlatList,
+	Image,
+	TouchableOpacity,
+	TextInput,
+} from "react-native";
+import { ImageData } from "./ImageData";
+
+const imageData: ImageData[] = [];
+for (let i = 1; i < 70; i++) {
+	imageData.push({ id: i, url: `https://picsum.photos/id/${i}/200` });
+}
 
 export default function App() {
-  return (
-    <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
-  );
+	const [displayLgImg, setDisplayLgImg] = useState(false);
```

Comment:
I think you can remove this piece of state on line 19 and use `featuredImgUrl` to hide/show the featured image. When you want to show the image, the string should not be blank, and when you hide it you can just reset your state to `""`